apps/ledger-live-desktop/src/renderer/families/bitcoin/types.ts

```diff
@@ -5,8 +5,14 @@ import {
 } from "@ledgerhq/live-common/families/bitcoin/types";
 import { LLDCoinFamily } from "../types";
 import { AccountDescriptor } from "@ledgerhq/live-common/families/bitcoin/descriptor";
+import { Operation } from "@ledgerhq/types-live";
```

Comment:
Just as you did for other families I would create a BitcoinOperation like this

```typescript
// in live-common/families/bitcoin/types.ts
import { Operation } from "@ledgerhq/types-live;
export type BitcoinOperation = Operation;
```

This way if we want for some reason to update the BitcoinOperation model the changes will be reflected here too + consistency